# SPDX-FileCopyrightText: Huawei Inc.
#
# SPDX-License-Identifier: Apache-2.0

Patch for //drivers/framework git repository of OpenHarmony 3.0 codebase.

Using //oniro/sysroots/target:target_clang toolchain, we now get a
working input-event-codes.h from the kernel headers, so we can drop
the dependency on the out-of-tree (FreeBSD!) header version.

Signed-off-by: Esben Haabendal <esben.haabendal@huawei.com>
Upstream-Status: Pending

diff --git a/drivers/framework/model/input/driver/event_hub.h b/drivers/framework/model/input/driver/event_hub.h
index 97dec781d33f..bd7a0f20a559 100644
--- a/drivers/framework/model/input/driver/event_hub.h
+++ b/drivers/framework/model/input/driver/event_hub.h
@@ -9,7 +9,7 @@
 #ifndef EVENT_HUB_H
 #define EVENT_HUB_H
 
-#include "input-event-codes.h"
+#include <uapi/linux/input-event-codes.h>
 #include "hdf_input_device_manager.h"
 #include "osal_time.h"
 
diff --git a/drivers/framework/model/input/driver/hdf_input_device_manager.h b/drivers/framework/model/input/driver/hdf_input_device_manager.h
index 98d247d88b91..f3d520755aa9 100644
--- a/drivers/framework/model/input/driver/hdf_input_device_manager.h
+++ b/drivers/framework/model/input/driver/hdf_input_device_manager.h
@@ -9,7 +9,7 @@
 #ifndef HDF_INPUT_DEVICE_MANAGER_H
 #define HDF_INPUT_DEVICE_MANAGER_H
 
-#include "input-event-codes.h"
+#include <uapi/linux/input-event-codes.h>
 #include "osal_mutex.h"
 #include "hdf_types.h"
 #include "hdf_device_desc.h"
